docs/api/config.md

```diff
@@ -252,6 +252,10 @@ By default, this is not enabled. You can enable this by setting it to `true`.
 
 This uses WebSockets instead of HTTP Server-Sent Events (SSE) as the transport protocol for UI updates. If you set this environment variable to `true`, then [`MESOP_CONCURRENT_UPDATES_ENABLED`](#MESOP_CONCURRENT_UPDATES_ENABLED) will automatically be enabled as well.
 
+### MESOP_APP_BASE_PATH
+
+This is the base path used to resolve other paths, particularly for serving static files. This is rarely needed because the default of using the current working directory is usually sufficient.
```

Comment:
Maybe mention that this needs to be an absolute path. I think sort of implied because it is a base path.

Reply:
good idea - done